lib/hacluster.pm

```diff
@@ -129,6 +132,20 @@ sub get_node_number {
     return script_output "crm_mon -1 | awk '/ nodes configured/ { print \$$index }'";
 }
 
+=head2 get_node_index
+
+ get_node_index();
```

Comment:
This `get_node_index` is finally not used in my PR, but I think it could be useful to keep it, WDYT?

Reply:
I vote to keep it.

Reply:
yes, it could be useful for something else.